spock-core/src/main/java/spock/lang/PendingFeatureIf.java

```diff
@@ -0,0 +1,51 @@
+package spock.lang;
+
+import groovy.lang.Closure;
+import org.spockframework.runtime.extension.ExtensionAnnotation;
+import org.spockframework.runtime.extension.builtin.PendingFeatureIfExtension;
+import org.spockframework.util.Beta;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * If a passed in closure evaluates to true, this will be treated as a {@link PendingFeature}
+ *
+ * Otherwise it will be treated like a normal feature.
+ *
+ * The closure follows the same rules as those used by {@link IgnoreIf}
```

Comment:
I would just copy the block with the information regarding the `PreconditionContext` from `@Requires`